packages/web3-eth/src/rpc_method_wrappers.ts

```diff
@@ -496,11 +496,16 @@ export async function getTransactionReceipt<ReturnFormat extends DataFormat>(
 		transactionHash,
 		DEFAULT_RETURN_FORMAT,
 	);
-	const response = await ethRpcMethods.getTransactionReceipt(
-		web3Context.requestManager,
-		transactionHashFormatted,
-	);
-
+	let response;
+	try {
+		 response = await ethRpcMethods.getTransactionReceipt(
+			web3Context.requestManager,
+			transactionHashFormatted,
+		);
+	} catch (error) {
+		console.warn(error)
```

Comment:
this will only warn instead of throw in other cases?, and thats not recommended

Reply:
Updated, thanks